res/controllers/Behringer-CMD-MM-1-Advanced-scripts.js

```diff
@@ -169,6 +169,15 @@ CMDMM.fxButton = function (channel, control, value, status, group) {
 		);
 	}
 };
+FUNCTIONS.prepFXUnitSettings = function () {
```

Comment:
Ah, sorry I didn't read the wiki close enough. I see now that the mapping does actually allow for controlling the enable buttons of the effects, so this isn't needed. I'll take care of it.

Reply:
Oh, I thought you had an issue with mixxx being out of sync with the FX buttons.